src/NUnitFramework/framework/Internal/Tests/Test.cs

```diff
@@ -289,11 +283,40 @@ public bool IsSuite
         /// <value></value>
         public int Seed { get; set; }
 
-        #endregion
+        /// <summary>
+        /// The SetUp methods.
```

Comment:
Since the other line is `The teardown methods`, I'd prefer to see `The setup methods` here.